Runtime/AvatarCreator/WebRequests/PartnerAssetsRequests.cs

```diff
@@ -69,7 +69,49 @@ public async Task<PartnerAsset[]> Get(BodyType bodyType, OutfitGender gender, Ca
             return assets.ToArray();
         }
 
-        public async Task<PartnerAsset[]> Get(Category? category, BodyType bodyType, OutfitGender gender, CancellationToken ctx = new CancellationToken())
+        public async Task<PartnerAsset[]> GetCategory(AssetType assetType, BodyType bodyType, OutfitGender gender, CancellationToken ctx = new CancellationToken())
+        {
+            var assets = new HashSet<PartnerAsset>();
```

Comment:
Maybe both the function(Get and GetCategory) in this class can be merge into one?

Reply:
yeah I actually don't remember why I did this 😅